### PR TITLE
Fuzzing test

### PR DESCRIPTION
**Pull request by Soreloser2**
Currently only one fuzzing test and that is to repeatedly press and unpress control and q over 100 tabs to see if anything errors. It's hard to really detect errors other than the page not being there but I suppose it's something.

Kind of takes a while to run (~50 seconds), maybe we run this as a chron job instead?

Would love other ideas. I was thinking about one that would visit random websites but there's no good way to do that afaik.

### CODE REVIEW

**Comment by paulstn** [comment]
We should have fuzzing tests with different types of websites. Lets do with with a new google doc (if possible), some new tabs, chrome://history, some websites that use content scripts, etc. etc.

**Comment by Soreloser2** [comment]
I'll write some fuzzing tests for a pre-set array of websites. 

I think google docs will be tricky, as it works on `docs.google.com`, but in order to get the bug related in #46 we would need to sign in and open a document to edit.

Also one thing to note is that the key input commands don't trigger the chrome commands api (as the keybinds aren't set), and so the chrome:// urls would have no behavior. We can still do a fuzzer test to make sure things don't break, but we can't really expect a changed tab.

**Review by riteshsb — APPROVED** [review]

File: test/integration/test-fuzzer.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    // not invalidate the state in any way.
+    await t.test("Test Random open and switch tabs", async (t) => {
+        var qdown = false;
+        var ctrlDown = false;
+        const iters = 1000;
+        for (var i = 0; i < iters; i++) {
+            await sleep(5);
+            var page = await getActivePage(browser);
+            assert.notStrictEqual(page, null);
+            var choice = Math.floor(Math.random() * 4);
```
Comment (paulstn) [inline test/integration/test-fuzzer.js:38]: Unless I'm mistaken, choice would be a random number that is either 0, 1, 2, or 3. But it seems like there's only 3 different choices being chosen from?

File: test/integration/test-fuzzer.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            } else if (choice === 1) {
+                // toggle ctrl being held down
+                if (ctrlDown) {
+                    await page.keyboard.up("Control");
+                    ctrlDown = false;
+                } else {
+                    await page.keyboard.down('Control');
+                    ctrlDown = true;
+                }
+            } else if (choice === 1) {
```
Comment (paulstn) [inline test/integration/test-fuzzer.js:57]: The conditional here is the same as what's in line 48, is this a typo?

**Review by paulstn — COMMENTED** [review]
I think this PR is great! Thank you for moving the getActivePage function to utils as well. I just have a few comments that I wanted to get addressed.

File: test/integration/test-fuzzer.js (hunk 1 above)
Comment (Soreloser2) [reply test/integration/test-fuzzer.js:38]: Oops yep!

**Review by paulstn — APPROVED** [review]
Looks good now!

**Pull request merged**